src/calliope_pathways/math/pathways.yaml

```diff
@@ -317,31 +317,43 @@ global_expressions:
     foreach: [nodes, techs, carriers, costs, investsteps]
     equations:
       - where: flow_cap_new
-        expression: sum($cost_sum * flow_cap_new, over=vintagesteps)
+        expression: sum($cost_sum * flow_cap_new * available_vintages, over=vintagesteps)
 
   cost_investment_storage_cap:
     foreach: [nodes, techs, costs, investsteps]
     equations:
       - where: storage_cap_new
-        expression: sum(cost_storage_cap * storage_cap_new, over=vintagesteps)
+        expression: sum(cost_storage_cap * storage_cap_new * available_vintages, over=vintagesteps)
 
   cost_investment_source_cap:
     foreach: [nodes, techs, costs, investsteps]
     equations:
       - where: source_cap_new
-        expression: sum(cost_source_cap * source_cap_new, over=vintagesteps)
+        expression: sum(cost_source_cap * source_cap_new * available_vintages, over=vintagesteps)
 
   cost_investment_area_use:
     foreach: [nodes, techs, costs, investsteps]
     equations:
       - where: area_use_new
-        expression: sum(cost_area_use * area_use_new, over=vintagesteps)
+        expression: sum(cost_area_use * area_use_new * available_vintages, over=vintagesteps)
 
   cost_investment_purchase:
     foreach: [nodes, techs, costs, investsteps]
 
   cost_investment:
     foreach: [nodes, techs, costs, investsteps]
+    equations:
+      - expression: >
+          $annualisation_weight * (
+            $depreciation_rate * (
```

Comment:
Similarly, could you explain where `annualisation_weight` and `depreciation_rate` are set?

Reply:
Looking at the expression, it seems we are close to [TEMOA's approach](https://temoacloud.com/temoaproject/Documentation.html#equation-obj-loan), which I think is good...

But honestly it's a tad hard to be sure on my part...

Reply:
It is defined in: https://github.com/calliope-project/calliope/blob/872978dfe3a305c85e0c279925fb794d970bf6bd/src/calliope/math/base.yaml#L871